test/e2e/nsx_subnet_test.go

```diff
@@ -36,17 +35,17 @@ func verifySubnetSetCR(subnetSet string) bool {
 	vpcNetworkConfig, err := testData.crdClientset.CrdV1alpha1().VPCNetworkConfigurations().Get(context.TODO(), vpcNetworkConfigCRName,
 		v1.GetOptions{})
 	if err != nil {
-		log.Printf("Failed to get VPCNetworkConfiguration %s: %v", vpcNetworkConfigCRName, err)
+		log.Info("Failed to get VPCNetworkConfiguration %s: %v", vpcNetworkConfigCRName, err)
```

Comment:
Could you also check this file? There are still some logs not in key value pattern.